Lib/test/libregrtest/main.py

```diff
@@ -626,8 +626,13 @@ def getloadavg_1m():
         elif sys.platform == 'win32' and (self.ns.worker_args is None):
             from test.libregrtest.win_utils import WindowsLoadTracker
 
-            load_tracker = WindowsLoadTracker()
-            self.getloadavg = load_tracker.getloadavg
+            try:
+                load_tracker = WindowsLoadTracker()
+                self.getloadavg = load_tracker.getloadavg
+            except FileNotFoundError as error:
+                # Windows IoT Core and Windows Nano Server do not provide
+                # typeperf.exe for x64, x86 or ARM
+                print('Failed to create WindowsLoadTracker: {}'.format(str(error)))
```

Comment:
The `str()` call here is redudant, an empty format specifier is essentially the same as calling `str()`

> A general convention is that an empty format string ("") produces the same result as if you 
> had called str() on the value. A non-empty format string typically modifies the result.

https://docs.python.org/3.4/library/string.html#format-specification-mini-language

Reply:
Thanks!